_data/sites.json

```diff
@@ -1072,6 +1082,16 @@
         ]
     },
 
+   {
+        "name": "AZMVDNOW",
+        "url": "https://azmvdnow.gov/customers/useraccount/disable",
+        "difficulty": "easy",
+        "notes": "Click ‘Disable Account’ button.  Check ‘Yes, I wish to disable my AZ MVD Now account.’.  Click ‘Disable Account’",
```

Comment:
Does it disable or delete account? That is a very nuanced difference, because on the former your data remains there, but on the latter everything related to you is wiped.

Reply:
Yes, but when it is disabled (without deleting your presence), we typically put it as impossible, after all it was not deleted, right?